pkg/hs/urbit-noun-core/lib/Urbit/Noun/Convert.hs

```diff
@@ -88,13 +88,13 @@ fromNoun :: FromNoun a => Noun -> Maybe a
 fromNoun n = runParser (parseNoun n) [] onFail onSuccess
   where
     onFail p m  = Nothing
-    onSuccess x = Just x
+    onSuccess x = Just $ force x
```

Comment:
can't hurt: do `onSuccess !x` and on the other two.

we also need a convention that everything we FromNoun have strict fields.